src/fides/api/schemas/saas/saas_config.py

```diff
@@ -194,8 +194,8 @@ def validate_grouped_inputs(cls, values: Dict[str, Any]) -> Dict[str, Any]:
 
         return values
 
-    @root_validator
-    def validate_override(cls, values: Dict[str, Any]) -> Dict[str, Any]:
+    @root_validator(allow_reuse=True)
```

Comment:
nit: does this really need the `allow_reuse=True`? i thought this was used if you actually wanted to reuse the same function/method for multiple validators; in your case here, you're defining a new `validate_request` validator function in the `ReadSaaSRequest` now below, right?